pkg/evm/evm.go

```diff
@@ -660,38 +660,17 @@ func SetupProposerVM(
 	if err != nil {
 		return err
 	}
-	return IssueTxsToActivateProposerVMFork(client, chainID, privKey)
-}
-
-func IssueTxsToActivateProposerVMFork(
-	client ethclient.Client,
-	chainID *big.Int,
-	privKey *ecdsa.PrivateKey,
-) error {
 	var errorList []error
-	var err error
 	for i := 0; i < repeatsOnFailure; i++ {
-		ctx, cancel := utils.GetAPILargeContext()
-		defer cancel()
-		err = issueTxsToActivateProposerVMFork(client, ctx, chainID, privKey)
+		err = issueTxsToActivateProposerVMFork(client, chainID, privKey)
 		if err == nil {
 			break
 		}
-		err = fmt.Errorf(
-			"failure issuing txs to activate proposer VM fork for client %#v: %w",
-			client,
-			err,
-		)
+		err = fmt.Errorf("failure issuing tx to activate proposer VM: %w", err)
```

Comment:
nit: add msg that it will retry.
also you can display to user that it's retry X out of Y

Reply:
this is not an error that will be displayed rt, but afterwards all repetitions failed (so retry msg not needed).
the idea is to avoid cluttering the user with the same error on the many repetitions. if adding X/Y it will also be considered a different message and will clutter output. keeping as it is for the moment.